### DISCUSSION TITLE
Clickable URLs in listViews, itemViews, descriptions

### DISCUSSION THREAD

**Discussion by jschuur** [keystonejs/keystone#7822] [Feature requests]
I have a text field in one of my lists that contains a URL, and it would be useful to make that URL clickable, even if you're just clicking on the link with no custom link text:

<img width="929" alt="CleanShot 2022-08-22 at 09 45 44@2x" src="https://user-images.githubusercontent.com/92457/185879744-4b5e731e-0e33-4e5a-b3b4-2f723378f521.png">

Tricker obviously would be in an itemView if the link is still editable there. Perhaps via a clickable link under the input box?

<img width="603" alt="CleanShot 2022-08-22 at 09 46 16@2x" src="https://user-images.githubusercontent.com/92457/185879735-ab55e476-dd27-40dd-9330-037a48cc0172.png">

I will take a look at how this might be done via [custom fields](https://keystonejs.com/docs/guides/custom-fields) in the mean time.

Related, it would be nice if the `ui.description` field contain linked text (perhaps just make that field use markdown?). In this example, I wanted to the 'git repo' text to a specific GitHub repo:

<img width="600" alt="CleanShot 2022-08-22 at 09 50 59@2x" src="https://user-images.githubusercontent.com/92457/185881050-b17d7718-a4d6-4195-88e6-d8b06cd96455.png">

**Comment by jschuur** [comment]
Looking at the Custom Fields documentation It should be pretty easy to hyperlink a field in a list view via a [Cell](https://keystonejs.com/docs/guides/custom-fields#cell) e.g..

**Comment by ThomasVille** [comment]
We also have a use case for showing links inside `ui.description`. One of our fields must contain a [SemVer](https://semver.org/) range, so it would be great if we could add a link to the documentation page, right in the description.

Allowing `ui.description` to contain HTML, Markdown or even just a React component would solve this issue.
